integrated_channels/integrated_channel/exporters/learner_data.py

```diff
@@ -253,27 +254,12 @@ def export(self, **kwargs):  # pylint: disable=R0915
             channel_name=channel_name,
             enterprise_customer_identifier=self.enterprise_customer.name
         )
-        enrollment_queryset = EnterpriseCourseEnrollment.objects.select_related(
```

Comment:
it was not removed, just refactored into a separate method for better readability in this method, the `create_enrollment_queryset` was extracted from the removed blocks of code here

Reply:
another related reason was I wanted to simplify what happens in the for loop that follows, so now that loop runs on a pre filtered set, rather than doing filtering within the loop